torch/distributed/nn/api/remote_module.py

```diff
@@ -210,109 +211,109 @@ def remote_parameters(self, recurse: bool = True) -> List[rpc.RRef[Parameter]]:
+    def apply(self: T, fn: Callable[["Module"], None]) -> T:  # type: ignore[return]
         _raise_not_supported(self.apply.__name__)
 
-    def cuda(self: T, device: Optional[Union[int, device]] = None) -> T:
+    def cuda(self: T, device: Optional[Union[int, device]] = None) -> T:  # type: ignore[return]
         _raise_not_supported(self.cuda.__name__)
 
-    def cpu(self: T) -> T:
+    def cpu(self: T) -> T:  # type: ignore[return]
         _raise_not_supported(self.cpu.__name__)
 
-    def type(self: T, dst_type: Union[dtype, str]) -> T:
+    def type(self: T, dst_type: Union[dtype, str]) -> T:  # type: ignore[return]
         _raise_not_supported(self.type.__name__)
 
-    def float(self: T) -> T:
+    def float(self: T) -> T:  # type: ignore[return]
         _raise_not_supported(self.float.__name__)
 
-    def double(self: T) -> T:
+    def double(self: T) -> T:  # type: ignore[return]
         _raise_not_supported(self.double.__name__)
 
-    def half(self: T) -> T:
+    def half(self: T) -> T:  # type: ignore[return]
         _raise_not_supported(self.half.__name__)
 
-    def bfloat16(self: T) -> T:
+    def bfloat16(self: T) -> T:  # type: ignore[return]
         _raise_not_supported(self.bfloat16.__name__)
 
-    def to(self, *args, **kwargs):
+    def to(self, *args, **kwargs):  # type: ignore[return]
         _raise_not_supported(self.to.__name__)
 
-    def register_backward_hook(
+    def register_backward_hook(  # type: ignore[return]
         self, hook: Callable[["Module", _grad_t, _grad_t], Union[None, Tensor]]
     ) -> RemovableHandle:
         _raise_not_supported(self.register_backward_hook.__name__)
 
-    def register_forward_pre_hook(self, hook: Callable[..., None]) -> RemovableHandle:
+    def register_forward_pre_hook(self, hook: Callable[..., None]) -> RemovableHandle:  # type: ignore[return]
         _raise_not_supported(self.register_forward_pre_hook.__name__)
 
-    def register_forward_hook(self, hook: Callable[..., None]) -> RemovableHandle:
+    def register_forward_hook(self, hook: Callable[..., None]) -> RemovableHandle:  # type: ignore[return]
         _raise_not_supported(self.register_forward_hook.__name__)
 
-    def state_dict(self, destination=None, prefix="", keep_vars=False):
+    def state_dict(self, destination=None, prefix="", keep_vars=False):  # type: ignore[return]
         _raise_not_supported(self.state_dict.__name__)
 
     def load_state_dict(
         self,
         state_dict: Union[Dict[str, Tensor], Dict[str, Tensor]],
         strict: bool = True,
-    ):
+    ):  # type: ignore[return]
         _raise_not_supported(self.load_state_dict.__name__)
 
     def parameters(self, recurse: bool = True) -> Iterator[Parameter]:
         raise ValueError(
             "Method ``parameters`` not supported for RemoteModule. Please use ``remote_parameters`` instead."
         )
 
-    def named_parameters(
+    def named_parameters(  # type: ignore[return]
```

Comment:
multiline type ignore is fixed a while back. we can put all these at the end of a AST scope

Reply:
Sorry I can't reproduce it locally (after rebasing and recompiling on master). Do I also need to update the mypy version with conda?

Reply:
Moving the type ignore to the end of a multiline statement actually fails the test: https://app.circleci.com/pipelines/github/pytorch/pytorch/237185/workflows/9d10924c-84e9-471b-a9ae-ffe3d6524a7a/jobs/8817770. Is it because in circleci we are using an old version of mypy?

Reply:
This feature is supported by only Python > 3.8, so we don't support it here.